bindings/rust/s2n-tls/src/client_hello.rs

```diff
@@ -168,6 +168,27 @@ impl ClientHello {
         Ok(session_id)
     }
 
+    fn server_name(&self) -> Result<Vec<u8>, Error> {
```

Comment:
Should this return a String instead of a Vec<u8>? I'm still not sure which one is better.